WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -170,6 +170,12 @@ private extension ItemListView {
             } label: {
                 SimpleProductCardView(product: simpleProduct)
             }
+        case .parentProduct(let parentProduct):
+            Button {
+                print("tapped parent product")
```

Comment:
Since we now treat POS as released, we should ensure that these half-baked states don't get into production. 

I know that for now variable products only appear for unreleased Woo versions but maybe we should hide them behind the feature flag to feel safe during the development. What are your thoughts on this?

Reply:
Yes, variable products are only loaded remotely when the local feature flag is enabled (disabled in production):

https://github.com/woocommerce/woocommerce-ios/blob/7c42ba8fd15211f43b9c0e93ab4a3fbd64acbed4/Yosemite/Yosemite/PointOfSale/PointOfSaleProductService.swift#L44-L46

Therefore, variable parent products do not show up in the item selector in POS regardless of their WC version.

Reply:
Ah, great!